README.md

```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
 #### Debian-based distro:
 
     sudo apt-get install nodejs git build-essential python2.7
+    #Raspberry Pi Cjdns Install
+
+##### Raspberry Pi
+###### Debian-Based
+Works on Raspbian, OSMC, Kali ARM Raspberry pi
+
+Your gonna need these packages: `
+apt-get install nodejs build-essential git`
+
+Next, grab the repository and build.
+
+    cd /opt
+    git clone https://github.com/cjdelisle/cjdns.git
+    cd cjdns
+    NO_TEST=1 Seccomp_NO=1 ./do`
```

Comment:
The CFLAGS are necessary for some 10+ times performance gain if I remembered correctly: https://github.com/tomeshnet/prototype-cjdns-pi2/blob/master/scripts/install#L8-L28